<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddServer</class>
 <widget class="QDialog" name="AddServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>200</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>325</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>325</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add new Server</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Server:</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_Server</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Server">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>28</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>54</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>User:</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_User</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_User">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>73</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>99</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_Password</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Password">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>118</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Port">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>144</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
   <property name="buddy">
    <cstring>spinBox_Port</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Port">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>163</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>8080</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Server</tabstop>
  <tabstop>lineEdit_User</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>spinBox_Port</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddServer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
